S/SDPA/build_tarballs.jl

```diff
@@ -68,15 +72,15 @@ if [[ $target == *"apple-darwin"* ]]; then
   macos_extra_flags="-DCMAKE_CXX_COMPILER_ID=AppleClang -DCMAKE_CXX_COMPILER_VERSION=10.0.0 -DCMAKE_CXX_STANDARD_COMPUTED_DEFAULT=11"
 fi
 
-cmake $macos_extra_flags -DCMAKE_INSTALL_PREFIX=${prefix} \
+cmake $macos_extra_flags \
+      -DCMAKE_FIND_ROOT_PATH=${prefix} \
+      -DCMAKE_INSTALL_PREFIX=${prefix} \
       -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DJulia_PREFIX=${prefix} \
       -DSDPA_DIR=$prefix \
       -DMUMPS_INCLUDE_DIR="../../destdir/include/mumps_seq" \
-      -DCMAKE_FIND_ROOT_PATH=${prefix} \
-      -DJulia_PREFIX=${prefix} \
       -DSDPA_LIBRARY="-lsdpa" \
       -D_GLIBCXX_USE_CXX11_ABI=1 \
-      -DJlCxx_DIR=${prefix}/lib/cmake/JlCxx \
```

Comment:
I took the liberty of sorting these arguments alphabetically and generally aligning them with other JLLs, in the hope that this will help a bit with future maintenance.